### PR TITLE
feat(ui): TE-2506 add notification configuration step in new simple create alert flow

### PR DESCRIPTION
**Pull request by nalin-patidar**
#### Issue(s)

https://startree.atlassian.net/browse/TE-2506

#### Description

In the new simple alert create flow, show the notification configuration step as part of base flow only rather then showing a modal once the alert is created.

### CODE REVIEW

File: thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-easy-page/alerts-create-easy-page.styles.ts [hunk 1]
```diff
@@ -134,6 +134,10 @@ export const easyAlertStyles = makeStyles((theme) => ({
             borderWidth: "1px",
         },
     },
+    notifcationContainer: {
```
Comment (cyrilou242) [inline thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-easy-page/alerts-create-easy-page.styles.ts:137]: `notifcationContainer`

typo 

`notificationContainer`

File: thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-base-page.component.tsx [hunk 2]
```diff
@@ -75,12 +75,12 @@ export const AlertsCreateBasePage: FunctionComponent<AlertsCreatePageProps> = ({
                     [QUERY_PARAM_KEY_ANOMALIES_RETRY, "true"],
                     [QUERY_PARAM_KEY_ALERT_TYPE, "create"],
                 ]);
-                if (isNewCreateEasyAlert) {
-                    searchParamsToSet.set(
-                        QUERY_PARAM_KEY_FOR_NOTIFICATION,
-                        "true"
-                    );
-                }
+                // if (isNewCreateEasyAlert) {
```
Comment (cyrilou242) [inline thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-base-page.component.tsx:78]: it's ok to delete right ?

**Review by cyrilou242 — APPROVED** [review]
LGTM, added minor style notes

File: thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-base-page.component.tsx (hunk 2 above)
Comment (nalin-patidar) [reply thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-base-page.component.tsx]: yes, done

File: thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-easy-page/alerts-create-easy-page.styles.ts (hunk 1 above)
Comment (nalin-patidar) [reply thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-easy-page/alerts-create-easy-page.styles.ts]: done

**Pull request merged**